site/docs/foundations/typography.mdx

```diff
@@ -5,101 +5,102 @@ sidebar:
   priority: 1
 ---
 
-Typography involves the style and appearance of written language. It plays a crucial role in enhancing readability, establishing hierarchy, and setting a visual tone, resulting in more effective and expressive communication.
+Typography refers to the predefined typographic styles that guide the presentation of written language. Typography is essential for improving readability, creating a clear hierarchy, and establishing a visual tone, all of which contribute to more effective and expressive communication. Each text style in Salt includes a font family, font size, line height and font weight. The specific values for each is dependent on the applied theme.
 
 ## Font family
 
-The Legacy UITK theme uses the open-source, sans-serif typeface **Open Sans**. The JPM brand theme uses the **Amplitude** font which is in line with updated brand guidelines. Code uses the mono space typeface **PT Mono**.
-
-For more information about the themes offered in Salt, please refer to the guidance on the [themes page](https://www.saltdesignsystem.com/salt/themes/index).
+The typography foundation offers two font families for UI design: Open Sans and Amplitude, along with PT Mono for code blocks.
 
 <Image
-  src="/img/foundations/typography-options.png"
-  alt="Example of Open Sans and Amplitude font"
+  src="/img/foundations/typography-fontfamily.png"
+  srcDark="/img/foundations/typography-fontfamily-dark.png"
+  alt=""
+  aria-hidden="true"
 />
 
-To download the Amplitude font, please refer to the [font brand resources](https://go/salt-ds-internal/resources/index) on our internal site. (Internal users only)
+Font families are used differently by each theme. For more information about the themes offered in Salt, please refer to [theme guidance](https://www.saltdesignsystem.com/salt/themes/index).
 
-## Font weight
+Each theme uses font families differently:
 
-Weight refers to the relative thickness of a font’s stroke. Each type style includes three weights which enable you to create visual hierarchy within the same text size.
+1. JPM Brand
```

Comment:
Can we make this an unordered list with the bullets indented? e.g. 
- JPM Brand
  - Amplitude - heading, display and action styles
  - Open Sans - body, label and notation styles